sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -460,18 +460,13 @@ AZ_NODISCARD az_result az_iot_hub_client_twin_get_publish_topic_get(
  *
  * @param[in] client The #az_iot_hub_client to use for this call.
  * @param[in] request_id The request id.
- * @param[in] if_match_version Can be either "*" to overwrite or the twin version to limit the

```

Comment:
Are these not part of the topic? Can they only be accessed via the payload?

Reply:
I confirmed with Ewerton this is not for MQTT or AMQP. When I did a search in the current SDK it only came up in HTTP.